content/docs/development/crds/carto.run_deliverables.yaml

```diff
@@ -62,12 +62,13 @@ status:
     - # Condition contains details for one aspect of the current
       # state of this API Resource. --- This struct is intended
       # for direct use as an array at the field path
-      # .status.conditions.  For example, type FooStatus struct{
-      # // Represents the observations of a foo's current state.
-      # // Known .status.conditions.type are: "Available",
-      # "Progressing", and "Degraded" // +patchMergeKey=type //
-      # +patchStrategy=merge // +listType=map // +listMapKey=type
-      # Conditions []metav1.Condition `json:"conditions,omitempty"
+      # .status.conditions.  For example, 
```

Comment:
I wasn't sure if this was introducing some meaningful change (live `cartographer.sh` on the right)

<img width="1381" alt="image" src="https://user-images.githubusercontent.com/3574444/194124402-fb6ea4cf-e964-4fec-9896-f001ee9e04a1.png">

was this block meant to be commented out? wondering if it's something weird going on with the generator

Reply:
Yeah, things get weird when processing arrays of structs. I'll see if there's any quick workaround for this one...